webapp/locales/en.json

```diff
@@ -102,7 +102,7 @@
       "projects": "Projects",
       "invites": "Invites",
       "follows": "Follows",
-      "shouts": "Shouts"
+      "shouts": "Recommended"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-07T19:13:34Z" title="Friday, June 7th 2019, 9:13:34 pm +02:00">Jun 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -102,7 +102,7 @@
       "projects": "Projects",
       "invites": "Invites",
       "follows": "Follows",
-      "shouts": "Shouts"
+      "shouts": "Recommended"
```


I don't think we need to change this in English

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-07T22:39:16Z" title="Saturday, June 8th 2019, 12:39:16 am +02:00">Jun 8, 2019</time>_

---


@mattwr18 do you think "Shouting" could be misunderstood in a negative way? As I pointed out in the comment above :point_up:?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-13T22:46:35Z" title="Friday, June 14th 2019, 12:46:35 am +02:00">Jun 14, 2019</time>_

---


sorry, @roschaefer, I saw this, but didn't reply...
I don't think so, but maybe shout-out would be better... like to give someone or something a shout-out... just that I think Instagram already uses that